tsdb/db.go

```diff
@@ -600,6 +600,8 @@ func open(dir string, l log.Logger, r prometheus.Registerer, opts *Options, rngs
 			return
 		}
 
+		close(db.donec) // DB is never run if it was an error, so close this channel here.
```

Comment:
Missed that on my PR, sorry :grimacing: 

Do you mind explaining how exactly this and the other channels(`compactc` and `stopc`)  are used by the DB object?

Reply:
> Do you mind explaining how exactly this and the other channels(compactc and stopc) are used by the DB object?

I don't remember from top of my head but you could search for their occurrences in the file. I think `compactc` is to trigger a compaction and when `stopc` channel is closed it tells all components to stop its activities and exit.

Reply:
> Missed that on my PR, sorry

No issues :) we missed it in the reviews too